README.md

```diff
@@ -4,28 +4,12 @@
 
 # Mentor Match (API)
 
-## Introduction
-  Mentor Match is an app for internal staff and student use at Turing School of Software & Design. Incoming Turing students have the opportunity to be matched to a software development mentor, usually an alumni, and Iteration 1 of Mentor Match gives admin tools to manage the matching process.
-
-  Nearing graduation, students also contact alumni for short-term mentoring (mock interviews, coffee meetings) as a part of Turing's Professional Development curriculum. This app provides the ability to search for mentors in areas of particular interest.
```

Comment:
Did you want to cut this part out as well? Or maybe expand the description of the app later on?

Reply:
Yes, it's cut here because I reworded it below in the descriptions of each Iteration, seems like a better way to outline our process and the functionality. Are there any changes you would make below to the iteration descriptions?

Reply:
I see where the incorporation is in the description of the iterations, and I'm thinking it might be helpful to include the info in the brief summary in the beginning so that someone visiting the project can quickly understand what it's all about, then read through the details.

Reply:
I think the proposed order is quickly understandable by the reader because it gives the purpose in the "Introduction" and immediately afterward lists the pieces of functionality. I was finding the previous version of the introduction to be too much to read in paragraph form, so the new version is more of a "show, don't tell" by listing the steps of functionality and below that exhibiting functionality in video form. It may look jumbled in markdown form, though, and better once it's loaded. What do you think about merging this version and then letting me know what you think of the flow on the page?